jablib/src/main/java/org/jabref/logic/cleanup/CleanupWorker.java

```diff
@@ -10,18 +10,25 @@
 import org.jabref.model.FieldChange;
 import org.jabref.model.database.BibDatabaseContext;
 import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.BibEntryPreferences;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class CleanupWorker {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(CleanupWorker.class);
     private final BibDatabaseContext databaseContext;
     private final FilePreferences filePreferences;
     private final TimestampPreferences timestampPreferences;
+    private final BibEntryPreferences bibEntryPreferences;
     private final List<JabRefException> failures;
-
-    public CleanupWorker(BibDatabaseContext databaseContext, FilePreferences filePreferences, TimestampPreferences timestampPreferences) {
+    
```

Comment:
No additional spaces please

In IntelliJ this, is should be automatically removed; maybe you are using another IDE?